.circleci/config.yml

```diff
@@ -32,13 +34,52 @@ jobs:
           name: Authenticate with registry
           command: |
             echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
-            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc 
+            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
           name: Publish @pxblue/publish
           command: |
             cd pxb-publish
             bash scripts/publish.sh -b $CIRCLE_BRANCH
 
+  publish_pxblue_tag:
+    docker:
+      - image: circleci/node:12.16.3-browsers
+    steps:
+      - checkout
+      - run:
+          name: Authenticate with registry
+          command: |
+            echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+            echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run:
+          name: Publish @pxblue/tag
+          command: |
+            cd pxb-tag
+            bash ../pxb-publish/scripts/publish.sh -b $CIRCLE_BRANCH
```

Comment:
It probably doesn't really matter too much, but would we want to use the published version of the publish command (via npx) for this instead of the local copy of the command?

Reply:
so like have the `publish:package` script in the package.json and then call `yarn publish:package -b $CIRCLE_BRANCH` here?

Reply:
I don't think it makes sense to use npx here because then you won't be using the "latest" (unpublished) code to publish.

Reply:
but this job is to 'officially publish the tag package'. I would expect all official publishing to be done with the official publish package on NPM (the exception being publishing the publish package because when you are doing that, presumably you've made changes in the way publishing works and you'll want to use the local copy).